package.json

```diff
@@ -30,13 +30,15 @@
     "mobile"
   ],
   "dependencies": {
+    "@types/color": "^3.0.0",
     "bplist-parser": "0.1.0",
     "bufferpack": "0.0.6",
     "byline": "4.2.1",
     "chalk": "1.1.0",
     "chokidar": "1.7.0",
     "cli-table": "https://github.com/telerik/cli-table/tarball/v0.3.1.2",
     "clui": "0.3.1",
+    "color": "^3.0.0",
```

Comment:
Strict version, please (you can use `npm i --save --save-exact color` in order to update package.json and npm-shrinkwrap.json correctly)